scripts/alerts-during-perf-testing.sh

```diff
@@ -13,7 +13,7 @@ RHSSO="rhsso"
 USER_SSO="user-sso"
 THREESCALE="3scale"
 TOKEN=$(oc whoami --show-token)
-MONITORING_ROUTE=$(echo "https://$(oc get route prometheus-route -n ${NAMESPACE_PREFIX}middleware-monitoring-operator -o=jsonpath='{.spec.host}')")/api/v1/alerts
+MONITORING_ROUTE=$(echo "https://$(oc get route prometheus-route -n ${NAMESPACE_PREFIX}observability -o=jsonpath='{.spec.host}')")/api/v1/alerts
```

Comment:
The route name will probably change with https://issues.redhat.com/browse/MGDAPI-2717

Reply:
Oh okay, good to know, so I think depending on which of these PRs merge first, will rebase / make the changes to take account for the route change